### PR TITLE
Replace yaz0 with orthrus, automatically [de]compress.

### PR DESCRIPTION
**Pull request by Cuyler36**
This PR replaces the older yaz0[.exe] tool with orthrus[.exe] (thanks @NWPlayer123). Orthrus includes the latest yaz0 compression optimizations. This allows us to automatically decompress & recompress foresta.rel.szs quickly.

I also added automation for decompressing foresta.rel.szs and recompressing it to remove a couple manual steps.

### CODE REVIEW

File: configure.py
Comment (Prakxo) [inline configure.py:1]: Tested from clean clone, and is it intended for the user to run configure after building to get the out rel compressed? Maybe it should be a build rule so it compresses after an OK

File: README.MD [hunk 1]
```diff
@@ -17,41 +17,37 @@ Use `--recursive` when cloning to have ppcdis in the repository.
...
-5. Download the [CodeWarrior 1.3.2 and 1.2.5n compilers](https://files.decomp.dev/compilers_20230715.zip) and extract them to *tools/1.3.2/* and *tools/1.2.5n/*, respectively.
-6. Download the [CodeWarrior 1.3.2r compiler](https://mega.nz/file/WuBFTCLT#TmB5R4-1mEFkk4G1Vjn9_cHXRD9wOIH9CtOLaVSWEas) and extract it to *tools/1.3.2r/*.
-7. Install Python, pip, and [ninja](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages#package-managers) using your package manager of choice.
-8. Install Python modules from requirements.txt (`pip install -r requirements.txt`).
-9. Install [wibo](https://github.com/decompals/wibo)
+1. [Dump a copy of the game](./docs/extract_game.md) and extract all files.
+2. Place **main.dol**, **foresta.rel.szs**, **forest_1st.arc**, and **forest_2nd.arc** in *dump/*.
+3. Download the [CodeWarrior 1.3.2 and 1.2.5n compilers](https://files.decomp.dev/compilers_20230715.zip) and extract them to *tools/1.3.2/* and *tools/1.2.5n/*, respectively.
+4. Download the [CodeWarrior 1.3.2r compiler](https://mega.nz/file/WuBFTCLT#TmB5R4-1mEFkk4G1Vjn9_cHXRD9wOIH9CtOLaVSWEas) and extract it to *tools/1.3.2r/*.
```
Comment (Prakxo) [inline README.MD:36]: Isn't this already in the [compilers](https://files.decomp.dev/compilers_latest.zip) from the server?

File: README.MD [hunk 2]
```diff
@@ -17,41 +17,37 @@ Use `--recursive` when cloning to have ppcdis in the repository.
...
+5. Install Python, pip, and [ninja](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages#package-managers) using your package manager of choice.
+6. Install Python modules from requirements.txt (`pip install -r requirements.txt`).
+7. Install [wibo](https://github.com/decompals/wibo)
     - Wibo is a lightweight Wine replacement that's tailor-made for use with decomp projects. Regular Wine can be used if preferred, but for the purposes of this guide, these instructions will use wibo.
     - Download [the latest GitHub release](https://github.com/decompals/wibo/releases/latest) and run `install ./wibo /usr/bin` to install it to your system.
```
Comment (Prakxo) [inline README.MD:41]: If wibo is broken we should say it here to not download the latest

**Review by Prakxo — COMMENTED** [review]
Looks good, just few things

File: README.MD (hunk 2 above)
Comment (Cuyler36) [reply README.MD:41]: You're right, I'll resolve these!

File: configure.py
Comment (Cuyler36) [reply configure.py:1]: It wasn't building when I had it as a rule for some reason. Maybe due to n.default call? I have no idea. That's why I moved it to subprocess.

**Review by Prakxo — APPROVED** [review]

**Pull request merged**